### ISSUE TITLE
Simplify default node for lineage graph

### ISSUE THREAD

**Issue by rjake** [dbt-labs/dbt-docs#214]
### Describe the feature
When a model is used heavily, the lineage chart is huge and freezes my browser (Edge). It would be nice to either have an separate and empty lineage page to start from or to limit the model to just `table` or `+table` instead of `+table+`

### Describe alternatives you've considered
I went to the details for a different model then opened the lineage graph and bookmarked that page.
```
...model/rare_table.....g_i=common_table         # bookmarked
...model/common_table...g_i=%2Bcommon_table%2B   # freezes
...model/common_table...g_i=common_table         # ideal
```

### Additional context
The only way I have been able to get to the lineage is to click here. On large models, it freezes my browser. Also, we host the docs through github pages.

![image](https://user-images.githubusercontent.com/14003618/132910696-3bf16438-cba2-4c38-9155-952d0f5f0f6f.png)

### Who will this benefit?
Any user looking to see model dependencies.

### Are you interested in contributing this feature?
Wish I could but it's over my head.

**Comment by rjake** [comment]
Thanks so much @jtcohen6 There are 15 upstream and 95 downstream nodes for this particular model.

**Comment by rarup1** [comment]
@jtcohen6  is there a fix in dbt 1.5 for this issue? We are seeing issues as a result of +node+ still... It has become unmanageable and unusable for some of the core models.

We are on the brink of breaking our projects up as a result but it would be nice if there was a simple feature to enable 1+node+1 else you get things like this:

<img width="1673" alt="image" src="https://github.com/dbt-labs/dbt-docs/assets/31633498/3e8cd3f1-5709-4085-8e5e-46f286920960">

**Comment by jtcohen6** [comment]
@rarup1 Unfortunately, this isn't something we're planning to solve for within `dbt-docs`. If your project is getting quite large, I don't think it's a bad idea to [consider splitting it up into multiple](https://github.com/dbt-labs/dbt-core/discussions/6725). And we are actively working on a more scalable experience for DAG visualization and metadata exploration within dbt Cloud.
